lms/templates/courseware/course_about.html

```diff
@@ -78,7 +80,83 @@
     if (hashParams.includes("contacted")) {
       $("#contacted-message").show();
     }
-  })(this);
+    $(".register").click(function(event) {
```

Comment:
@jcdyer, one more issue I found while testing the enrollment notification feature is that the mouse pointer keeps changing shapes when trying to click the 'Enroll' button of a course from its 'About' page. Not sure if that is due to some issue in the code below.